source/globalCommands.py

```diff
@@ -667,19 +667,17 @@ def script_toggleReportTables(self,gesture):
 
 	@script(
 		# Translators: Input help mode message for toggle report table row/column headers command.
-		description=_("Toggles on and off the reporting of table row and column headers"),
+		description=_("Cycle through the possible modes to report table row and column headers"),
 		category=SCRCAT_DOCUMENTFORMATTING
 	)
 	def script_toggleReportTableHeaders(self,gesture):
-		if config.conf["documentFormatting"]["reportTableHeaders"]:
-			# Translators: The message announced when toggling the report table row/column headers document formatting setting.
-			state = _("report table row and column headers off")
-			config.conf["documentFormatting"]["reportTableHeaders"]=False
-		else:
-			# Translators: The message announced when toggling the report table row/column headers document formatting setting.
-			state = _("report table row and column headers on")
-			config.conf["documentFormatting"]["reportTableHeaders"]=True
-		ui.message(state)
+		from config.configFlags import ReportTableHeaders

```

Comment:
Why is this imported lazily?

Reply:
It's imported lazily as done in more recent scripts of `globalCommands.py` such as `script_toggleScreenCurtain`. It seems relevant to avoid to have too much import statements at the top of this file when the imported variable will not be used elsewhere.

Let's wait for the opinion of NVAccess reviewer. I may put back the import at the top of the file if people prefer this way to do.

Reply:
@seanbudd what is your opinion on this lazy import?

Reply:
Since config is already imported at the top level, this should probably be moved out.
I believe the screen curtain case (and others) is more about avoid circular dependencies.